docs/tutorials/mastering.md

```diff
@@ -1,60 +1,651 @@
+
+- mgmt-br route metric value is higher than your default route's metric
+- core-br metric value is higher than your mgmt-br route's metric
+
+Change the metrics of SD-Core routes which are indicated with comments below, relatively to your default route's metric if required. 
+
+```console
+cat << EOF | sudo tee /etc/netplan/99-sdcore-networks.yaml
+# /etc/netplan/99-sdcore-networks.yaml
+network:
+  bridges:
+    mgmt-br:
+      addresses:
+        - 10.201.0.14/24
+      routes:
+        - to: default
+          via: 10.201.0.1
+          metric: 110 # Set the value higher than your default route's metric
+    access-br:
+      addresses:
+        - 10.202.0.14/24
+      routes:
+        - to: 10.204.0.0/24
+          via: 10.202.0.1
+    core-br:
+      addresses:
+        - 10.203.0.14/24
+      routes:
+        - to: default
+          via: 10.203.0.1
+          metric: 203 # Set the value higher than your mgmt-br route's metric
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+
+  version: 2
+EOF
+```
+
+Arrange the file permissions and apply the network configuration.
+
+```console
+sudo chmod 600 /etc/netplan/99-sdcore-networks.yaml
+sudo netplan apply
+```
+
+```{note}
+Applying new netplan configuration may produce warnings related to file permissions being too open. You may safely disregard them.
+```
+
+### Install and Configure LXD
+
+Install LXD:
+
+```console
+sudo snap install lxd
+```
+
+Create custom storage pool and LXD project:
+
+```console
+lxc storage create sdcore zfs size=150GiB
+lxc project create sdcore --config features.images=true --config features.networks=true --config features.networks.zones=true --config features.profiles=true --config features.storage.buckets=true --config features.storage.volumes=true
+```
+
+Create the configuration to initialize the LXD. If the LXD version is lower than 5.9, you should remove the `features.networks.zones` key from the project configuration.
+
+```console
+cat << EOF | sudo tee ~/preseed.yaml
+config: {}
+networks: []
+storage_pools:
+- config:
+    size: 150GiB
+    source: /var/snap/lxd/common/lxd/disks/sdcore.img
+    zfs.pool_name: sdcore
+  description: ""
+  name: sdcore
+  driver: zfs
+profiles:
+- config: {}
+  description: Default LXD profile
+  devices:
+    root:
+      path: /
+      pool: sdcore
+      type: disk
+  name: default
+projects:
+- config:
+    features.images: "true"
+    features.networks: "true"
+    features.networks.zones: "true" # Omit this if LXD version is lower than 5.9.
+    features.profiles: "true"
+    features.storage.buckets: "true"
+    features.storage.volumes: "true"
+  description: SD-Core project
+  name: sdcore
+EOF
+```
+
+```console
+lxd init --preseed < ~/preseed.yaml 
+```
+
+```{note}
+If you get an error which says that `features.networks.zones` is not supported, please remove that key and apply the configuration again.
+```
```

Comment:
Please see the comment about the placement of this note above.

Reply:
Removed note.